DNN Platform/Tests/DotNetNuke.Tests.Integration/DotNetNuke.Tests.Integration.csproj

```diff
@@ -91,7 +91,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="PetaPoco">
-      <HintPath>..\..\Components\PetaPoco\bin\PetaPoco.dll</HintPath>
+      <HintPath>..\..\Library\bin\PetaPoco.dll</HintPath>

```

Comment:
(Same as above)

Instead of referencing the Library\bin\PetaPoco.dll could we update the test project to use the nuget file. This will limit a dependency on the compiled dll and force NuGet to resolve the dependency which I believe is the best practice we should be following